parts/classes/class-script-loader.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/* ---------------------------------------------------------------------------------------------
```

Comment:
Can we use [docblock](https://www.phpdoc.org/git) style comments instead these big huge blocks?

We should use standard PHP practices.

Reply:
Afaik this format is the expected format for WordPress coding standards. It's the same format used in previous default themes.

Reply:
See https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#6-file-headers

Reply:
> Afaik this format is the expected format for WordPress coding standards. It's the same format used in previous default themes.

Yeah, previous themes had some issues that passed through because TRT wasn't involved 😄We're trying to polish this one out so that we can say: look at 2020 🙂